packages/tracing-internal/src/browser/request.ts

```diff
@@ -186,6 +186,8 @@ export function fetchCallback(
     return;
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const contentLength = (handlerData.response?.headers).get('content-length');
```

Comment:
m: We don't want to use optional chaining here, it increases bundle size - let's instead just use `&&`